pkgs/development/compilers/atasm/default.nix

```diff
@@ -1,19 +1,22 @@
 { lib
 , stdenv
-, fetchurl
-, unzip
+, fetchFromGitHub
 , zlib
 }:
 
 stdenv.mkDerivation rec {
   pname = "atasm";
-  version = "1.09";
+  version = "1.23";
 
-  src = fetchurl {
-    url = "https://atari.miribilist.com/${pname}/${pname}${builtins.replaceStrings ["."] [""] version}.zip";
-    hash = "sha256-26shhw2r30GZIPz6S1rf6dOLKRpgpLwrqCRZX3+8PvA=";
+  src = fetchFromGitHub {
+    owner = "CycoPH";
+    repo = "atasm";
+    rev = "V${version}";
+    hash = "sha256-U1HNYTiXO6WZEQJl2icY0ZEVy82CsL1mKR7Xgj9OZ14=";
   };
 
+  makefile = "Makefile";
```

Comment:
Isn't it the default?

Reply:
If the file is found yes, in this case no because the file is not at the root so it does not trigger the `make` call (that's why the derivation currently builds nothing)

Reply:
But it should point to an existing file nonetheless - like  `src/Makefile`, not a mere non-file string.
This is strange...

Reply:
It is set to `-C ./src/` in the preBuild phase.